### ISSUE TITLE
add `consistent-import` rule

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod-x#178]
### 🧩 Description

See https://github.com/marcalexiei/eslint-plugin-zod-x/issues/172

### ⚙️ Options

- syntax
  - Default: 'namespace'
  - Values
    - `namespace`
    - `named`

### 🧾 Example Configuration

```ts
{
  'zod/consistent-import': ['error', { syntax: 'named' | 'namespace' }],
}
```

### ✅ Valid Code Examples

```typescript
//   'zod/consistent-import': ['error', { syntax: 'named' }],
import { z } from 'zod';

//   'zod/consistent-import': ['error', { syntax: 'namespace' }],
import * as z from 'zod';
```

### ❌ Invalid Code Examples

```typescript
//   'zod/consistent-import': ['error', { syntax: 'named' }],
import * as z from 'zod';
import z from 'zod';

//   'zod/consistent-import': ['error', { syntax: 'namespace' }],
import { z } from 'zod';
import z from 'zod';
```

### ✒️ Additional context

- https://github.com/marcalexiei/eslint-plugin-zod-x/issues/172#issuecomment-3733713408
